### DISCUSSION TITLE
invalid_setup: The credential status can't be verified because there is no ICredentialStatusVerifier plugin installed.

### DISCUSSION THREAD

**Discussion by umutyorulmaz** [decentralized-identity/veramo#1341] [Q&A] [answered]
Hey there, I was trying to verify the credential I created using terminal, following the veramo documentation part called 'CLI tutorials'. Getting the error below:

invalid_setup: The credential status can't be verified because there is no ICredentialStatusVerifier plugin installed.

I couldnt find the plugin mentioned in the error. Any suggestions ? 

Thank you.

**Comment by mirceanis** [comment, marked as answer]
Thanks for signaling this.

There is a bit of a mismatch there. The `credentialStatus` property being set by the CLI is outdated.

Please choose no when being asked if the credential should be revokable in the CLI. It should produce fully verifiable credentials after that.

**Reply by umutyorulmaz** [reply]
Yes, it worked, thank you!
